frontend/demo/component/board/example-indicator.ts

```diff
@@ -24,9 +25,9 @@ export class ExampleIndicator extends LitElement {
         font-size: var(--lumo-font-size-xxs);
       }
 
-      .icon iron-icon {
-        --iron-icon-width: var(--lumo-font-size-xxs);
-        --iron-icon-height: var(--lumo-font-size-xxs);
+      .icon vaadin-icon {
+        --vaadin-icon-width: var(--lumo-font-size-xxs);
+        --vaadin-icon-height: var(--lumo-font-size-xxs);
```

Comment:
`vaadin-icon` doesn't have this CSS variable, use plain width/height instead
```suggestion
        width: var(--lumo-font-size-xxs);
        height: var(--lumo-font-size-xxs);
```